core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -597,6 +631,10 @@ class GroupCoordinator(val brokerId: Int,
 
 object GroupCoordinator {
 
+  val EmptyGroup = GroupSummary(NoState, NoProtocolType, NoProtocol, NoMembers)
```

Comment:
Following the above comment: if we want to return an empty group with no error code, probably making the state as "Dead" is better for client-side handling?